kano_greeter/newuser_view.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+# password_view.py
+#
+# Copyright (C) 2014 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
+#
+
+from gi.repository import Gtk
+from gi.repository import LightDM
+
+import os
+
+from kano.logging import logger
+from kano.gtk3.buttons import KanoButton, OrangeButton
+from kano.gtk3.heading import Heading
+from kano.gtk3.kano_dialog import KanoDialog
+
+from kano_greeter.last_user import set_last_user
+
+from kano_world.functions import login as kano_world_authenticate
```

Comment:
@skarbat Do we need to depend on `kano-profile` and if so can't we just use that window for logging in for consistency.